### DISCUSSION TITLE
A few issues/suggestions with copilot for visual studio 2022

### DISCUSSION THREAD

**Discussion by SmikeSix** [community/community#48544] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Hi, I recently started using Copilot and have some suggestions/questions.

I developed a voice coding plugin for people with health issues, that provides IntelliSense autocomplete to an AI voice controller, but I'm facing issues when using it with Copilot. My extension is not yet released.

I had Copilot version 1.3 installed, but I switched to 1.73.3 a few minutes ago. However, version 1.73.3 has performance issues, such as not triggering Copilot features at times, and the code it inserts is formatted oddly compared to versions 1.3 or 1.71.

It does fix the main issue I had before, which was that IntelliSense added whitespaces when triggering a hidden session. I fixed this by starting the session at the first real character, but prior versions 1.73.3 of Copilot seemed to trigger this too.

Basically, the latest version of Copilot often is slower when triggering inline suggestions and shows fewer long answers, making it less useful.

Is there a way to redirect Copilot's inline suggestion to another extension without displaying it inline? 

This would be amazing. I spent months creating the extension and also made a tooltip-like window next to the caret to display speak-able context. It tries not to cover other code or IntelliSense so it won't annoy me when coding. Although the UI is not super useful since you can see the code anyway. But .. it would probably be a great addition to display Copilots suggestions there. 

![ScreenXX](https://user-images.githubusercontent.com/2948601/221331027-e3b59752-ca69-487f-810f-fbd0b976a426.PNG)
